### PR TITLE
Remove ActiveAdmin

### LINKED ISSUE #13: Remove active admin
We're currently using a gem, [active_admin](https://activeadmin.info/), which is deemed to be more trouble than it's worth. We would like to remove this gem. In a future issue, we can spell out how to replace it with something more suited to our needs.

### PR DESCRIPTION
**Pull request by marlena-b**
### Describe your change in plain English.
Removes ActiveAdmin from the project. 
### Link to the issue
Fixes #13

### CODE REVIEW

**Review by kasugaijin — APPROVED** [review]
Looks good to me! See below for one suggestion before merging.

**Review by kasugaijin — COMMENTED** [review]
I think we can remove these also
`pin "chartkick", to: "chartkick.js"`
`pin "Chart.bundle", to: "Chart.bundle.js"`
in `config/importmap.rb`

these were used to make some graphs in active admin dashboard

**Comment by marlena-b** [comment]
Thank you @kasugaijin! I missed that, it's fixed now

**Comment by marlena-b** [comment]
Tests are failing but looks like they are unrelated to my changes (broken on main)

**Comment by edwinthinks** [comment]
Yup @marlena-b it was just fixed in main, I feel pretty good about merging this in
